deployability/modules/testing/tests/test_agent/test_install.py

```diff
@@ -82,7 +82,9 @@ def test_installation(wazuh_params):
 
     # Agent installation
     for agent_name, agent_params in wazuh_params['agents'].items():
-        WazuhAgent.perform_install_and_scan_for_agent(agent_params, agent_name, wazuh_params)
+        #WazuhAgent.perform_install_and_scan_for_agent(agent_params, agent_name, wazuh_params)
```

Comment:
Remove comment

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5324/commits/96c47257f1bca33c7f144285a4c7b63a8c10fbce